lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelTest.java

```diff
@@ -943,4 +943,78 @@ public void detach_success_callback_detached() throws AblyException {
 				ably.close();
 		}
 	}
+
+	/**
+	 * <p>
+	 * Validate publish will result in an error, when the channel is
+	 * in or moves to the FAILED state before the operation succeeds
+	 * </p>
+	 * <p>
+	 * Spec: RTL6c3
+	 * </p>
+	 *
+	 * @throws AblyException
+	 */
+	@Test
```

Comment:
> I am doing that on line 975

How is that possible? The thing we're expecting is that the publish call (https://github.com/gokhanbarisaker/ably-java/blob/channel-test-implicitattach/lib/src/test/java/io/ably/lib/test/realtime/RealtimeChannelTest.java#L972) throws an exception. How will it reach line 975?